src/actions/users.js

```diff
@@ -937,6 +937,27 @@ export function updateMe(user) {
     };
 }
 
+export function patchUser(userId, user) {
```

Comment:
Why are we passing the userId as an argument? I think the user object should have the id

Reply:
Yes! You are right. Fixed